app/mailers/candidate_mailer.rb

```diff
@@ -394,20 +394,6 @@ def decline_last_application_choice(application_choice)
     )
   end
 
-  def course_unavailable_notification(application_choice, reason)
-    @application_choice = application_choice
-    email_for_candidate(
-      application_choice.application_form,
-      subject: I18n.t!(
-        "candidate_mailer.course_unavailable_notification.subject.#{reason}",
-        course_name: application_choice.current_course_option.course.name_and_code,
-        provider_name: application_choice.current_course_option.course.provider.name,
-        study_mode: application_choice.current_course_option.study_mode.humanize.downcase,
-      ),
-      template_name: "course_unavailable_#{reason}",
-    )
-  end
-
```

Comment:
This email was removed in 2020 (all views). Let me know if there is any reason that I should keep it.